pkgs/development/python-modules/gdown/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, filelock
+, requests
+, tqdm
+, setuptools
+}:
+
+buildPythonApplication rec {
+  pname = "gdown";
+  version = "3.11.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p023812hh7w7d08njjsfn0xzldl4m73yx8p243yb2q49ypjl6nz";
+  };
+
+  propagatedBuildInputs = [ filelock requests tqdm setuptools ];
+
+  doCheck = false;
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    $out/bin/gdown --help > /dev/null
+  '';
```

buildPythonApplication's checkPhase maps to installCheckPhase on derviation, and checkPhase = true; is default for python packages
```suggestion
  checkPhase = ''
    $out/bin/gdown --help > /dev/null
  '';
```

---

Sure, I made the changes and updated. I kept the `doCheck = false;` there to skip running this module's test.